docs/2_QUICK_START_GUIDE.md

```diff
@@ -1,13 +1,229 @@
+
+1. Navigate to the root directory of your project and use the following command to copy the environment template file.
+
+    ```bash
+    cp .env.template .env
+    ```
+
+    ```bash
+    $ tree infra
+    ├───.env.template
+    ├───.env ## NEW FILE GENERATED FROM THE COMMAND
+    ├───modules
+    │   └───providers
+    │       ├───azure
+    │       │   ├───...
+    │       │   └───vnet
+    │       └───common
+    └───templates
+        ├───az-hello-world
+        │   └───test
+        └───backend-state-setup
+    ```
+
+1. Provide values for the environment values in `.env` which are required to authenticate Terraform to provision resources within your subscription.
+
+    ```bash
+    # Versioning
+    GO_VERSION=1.12.5
+    TF_VERSION=0.12.2
+    BUILD_BUILDID=1
+    # Subscription Values
+    ARM_SUBSCRIPTION_ID="<az-service-principal-subscription-id>"
+    ARM_CLIENT_ID="<az-service-principal-client-id>"
+    ARM_CLIENT_SECRET="<az-service-principal-auth-secret>"
+    ARM_TENANT_ID="<az-service-principal-tenant>"
+    ARM_ACCESS_KEY="<remote-state-storage-account-primary-key>"
+    # Azure Blob Storage Values
+    TF_VAR_remote_state_account="<tf-remote-state-storage-account-name>"
+    TF_VAR_remote_state_container="<tf-remote-state-storage-container-name>"
+    TF_WARN_OUTPUT_ERRORS=1
+    TF_CLI_ARGS=
+    ```
+
+1. Execute the following commands to set up the environment variables for your bash session:
+
+    ```bash
+    # these commands setup all the environment variables needed to run this template
+    DOT_ENV=<path to your .env file>
+    export $(cat $DOT_ENV | xargs)
+    ```
+
+1. Execute the following login command to configure your local Azure CLI.
+
+    ```bash
+    # This logs your local Azure CLI in using the configured service principal.
+    az login --service-principal -u $ARM_CLIENT_ID -p $ARM_CLIENT_SECRET --tenant $ARM_TENANT_ID
+    ```
+
+    ```bash
+    # Other helpful azure cli commands
+    az --version # Cobalt does not recommend a version but this command proves useful for troubleshooting the az cli
+    az account show # A command for validating the azure cli is correctly configured
+    az configure --defaults group='' # Use empty strings to clear default values for any sticky az properties like "group"
+    ```
+
+### **Step 4:** Initialize a Terraform Remote Environment
+
+A Terraform environment is primarily any json file generated by the `terraform init` and/or `terraform workspace new` commands. Initializing a Terraform *remote* environment (as opposed to a local one) effectively means to create an Azure blob storage json file within your existing Azure blob storage account and container listed in the [prerequisites](#23-prerequisites).
```

Comment:
The terminology looks off here:
 - A terraform "environment" is an old way to refer to a terraform workspace (https://www.terraform.io/docs/state/environments.html) but this is not a preferred term as of now
 - The terraform syntax is `HCL` and not `json` and it is not generated by any terraform command. The workspace artifacts are, in part, JSON but there are other non JSON artifacts. Not sure its worthwhile to specify the internal workings of TF here

```suggestion
Terraform maintains a [state](https://www.terraform.io/docs/state/index.html) file that tracks important information about the resources that it has provisioned. It is a critical component to Terraform and it is worth understanding, but this will not be covered here.

A best practice when using Terraform is to store state artifacts in the cloud (i.e., Azure Blob Storage or AWS S3). In this example, we will configure a new Terraform workspace that will be persisted on Azure Blob Storage.
```

Reply:
I cleared this up. Less is more! Thanks!